### ISSUE TITLE
variable "vim" undefined in python_cmdline.vim

### ISSUE THREAD

**Issue by xuesoso** [jalvesaq/vimcmdline#68]
Variable "vim" throws error when I try to execute in visual mode (v+<Space>). I commented out the lines where it calls for "vim.eval..." without dire consequence. May need definition for / removal of "vim" here
![vimcmdline_error](https://user-images.githubusercontent.com/15006047/57185344-56a37b80-6e7e-11e9-9b74-fece1dec2109.png)
.

**Comment by jalvesaq** [comment]
@Mizudera was the last contributor to edit this part of the code.

**Comment by DaanJG** [comment]
I get the same error when sending multiple lines to jupyter-console:
```
Error detected while processing function VimCmdLineSendSelection[8]..PythonSourceLines[13]..provider#python#Call:                                                                              
line   18:                                                                                                                                                                                     
Traceback (most recent call last):                                                                                                                                                             
  File "<string>", line 3, in <module>                                                                                                                                                         
NameError: name 'vim' is not defined
```
This happens when sending visually selected lines using \<Space\> or when using \<LocalLeader\>p for a paragraph. Nevertheless, the lines get evaluated normally and the results are available at the Jupyter console. Strangely, the are no errors when sending lines to the default plain Python interpreter.

The internet says the NameError is due to lack of Python support compiled into Vim or Neovim, but why then no errors when using python instead of jupyter-console?

I'm using NVIM v0.2.2 without Python support, Jupyter console 5.2.0 and Python 3.6.7, as available in Ubuntu 18.04.

EDIT: I can confirm that xuesoso's fix works.

**Comment by jalvesaq** [comment]
It should work now if `:pythonx` works, that is if Vim is compiled with either +python or +python3.
